api/app/serializers/v1/ingestion_message_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module V1
+  class IngestionMessageSerializer < ManifoldSerializer
+    include ::V1::Concerns::ManifoldSerializer
+
+    typed_attribute :kind, Types::String.meta(read_only: true)
+    attribute :payload
```

Comment:
I'm pretty sure we can do `typed_attribute :payload, Types::Any.meta(read_only: true)` here. FE can repurpose the code they used for processing responses from ActionCable to know how to handle the `payload` value based on the `kind`.

Additionally, add `typed_attribute :id, Types::Serializer::ID` in the serialized message so FE has something to track which message(s) they've seen / processed to display to the user.